oauthproxy.go

```diff
@@ -458,17 +458,19 @@ func (p *OAuthProxy) ErrorPage(rw http.ResponseWriter, code int, title string, m
 }
 
 //<!--COCO Begin-->
-// COCO: logout from cocostore
-//<!--COCO End-->
+// COCO: logout from coco
 func (p *OAuthProxy) CocostoreLogout(rw http.ResponseWriter, title string) {
```

Comment:
- The function should be renamed to CocoLogout as per our yesterday's discussion

Reply:
handled